Feature/deploy keys

prasannamahajan opened the pull request:
Changing deploy key

prasannamahajan committed:
adding deploy key of lfx-kit

Signed-off-by: prasanna.mahajan <mahajanprasannap@gmail.com>

prasannamahajan committed:
removing old deploy key

Signed-off-by: prasanna.mahajan <mahajanprasannap@gmail.com>

prasannamahajan committed:
Merge branch 'master' of github.com:communitybridge/easycla-api into feature/deploy-keys

dealako commented on .circleci/config.yml:
```diff
@@ -18,45 +18,14 @@ jobs:
           command: make setup
-      - run:
-          name: Download dependencies for lfx-kit swagger models
-          command: make deps
```
As I recall, this is needed before running the swagger generate so that we have the `lfx-kit` data models.

prasannamahajan replied:
Resolved.
Instead of seperately calling it. I added it in dependency for target "swagger" and "build"

dealako reviewed (changes requested):
One comment.

prasannamahajan committed:
Adding dep as dependency in Makefile for target swagger
Removing `make dep` from circleci config as dep is already dependency
for target build

Signed-off-by: prasanna.mahajan <mahajanprasannap@gmail.com>
